src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      class="card__img"
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27&quot; Retina<br>5K Monoblock (MNED2UA/A)"
+      width="160"
+      height="134"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27&quot; Retina<br>5K Monoblock (MNED2UA/A)
```

Comment:
You have used a `<br>` tag in `h3` heading. It's better to avoid using `<br>` tags in headings or paragraph text. Consider using CSS to control the design or layout such as `width` or `height` to enforce line breaks where necessary.